x-pack/plugins/security_solution/server/lib/telemetry/endpoint_metadata_telemetry.ts

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { ISavedObjectsRepository } from 'src/core/server';
+import { AgentMetadata } from './../../../../ingest_manager/common/types/models/agent';
+import { AGENT_SAVED_OBJECT_TYPE } from './../../../../ingest_manager/common/constants/agent';
+import { AgentSOAttributes } from '../../../../ingest_manager/common';
+
+export interface EndpointMetadataOSTelemtry {
+  name: string;
+  version: string;
+  count: number;
+}
+
+export interface EndpointPoliciesTelemetry {
+  malware: {
+    success: number;
+    warning: number;
+    failure: number;
+  };
+}
+
+export interface IngestManagerLocalMetadata extends AgentMetadata {
```

Comment:
So fleet stores this information based on the endpoints that are connected?

Reply:
Yep, successfully installed I believe